### PR TITLE
[stable/openiban] Add OpenIBAN chart

### PR DESCRIPTION
**Pull request by verwilst**
Signed-off-by: Bart Verwilst <bart@verwilst.be>

**What this PR does / why we need it**: Add OpenIBAN chart

### CODE REVIEW

File: stable/openiban/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "openiban.fullname" . }}
+  labels:
+    app: {{ template "openiban.name" . }}
```
Comment (davidkarlsen) [inline stable/openiban/templates/deployment.yaml:6]: use updated labels best practices: https://github.com/helm/helm/blob/master/docs/chart_best_practices/labels.md (for all resources)

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

File: stable/openiban/templates/deployment.yaml (hunk 1 above)
Comment (davidkarlsen) [reply stable/openiban/templates/deployment.yaml:6]: Let's see what comes out of https://github.com/helm/charts/issues/7550

**Comment by verwilst** [comment]
Could this be merged since #7550 is quite dormant.

File: stable/openiban/templates/deployment.yaml (hunk 1 above)
Comment (davidkarlsen) [reply stable/openiban/templates/deployment.yaml:6]: No response on that - so let's keep it

File: stable/openiban/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+appVersion: 1.0.0
+description: OpenIBAN is a self-hosted, free and open-source IBAN validation API.
+name: openiban
+version: 0.1.0
```
Comment (davidkarlsen) [inline stable/openiban/Chart.yaml]: go straight for 1.0.0 since we're in stable?

File: stable/openiban/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+appVersion: 1.0.0
```
Comment (davidkarlsen) [inline stable/openiban/Chart.yaml]: image is 1.0.1?

**Review by davidkarlsen — APPROVED** [review]

**Pull request merged**